primitiv/c/graph.cc

```diff
@@ -7,224 +7,193 @@
 #include <vector>
 
 #include <primitiv/graph.h>
-
 #include <primitiv/c/internal.h>
 #include <primitiv/c/graph.h>
 
 using primitiv::Node;
 using primitiv::Graph;
+using primitiv::c::internal::to_c_ptr;
+using primitiv::c::internal::to_cpp_ptr;
+using primitiv::c::internal::to_c_ptr_from_value;
 
 extern "C" {
 
 primitiv_Node *primitiv_Node_new() {
```

Comment:
This function may throw an exception (generally, the default `new` operator may *always* throw `bad_alloc` exception), and should return a `primitiv_Status`.